lib/SPIRVProducerPass.cpp

```diff
@@ -6192,6 +6224,22 @@ bool SPIRVProducerPassImpl::selectFromSameObject(Instruction *inst) {
   return false;
 }
 
+bool SPIRVProducerPassImpl::isPointerUniform(Value *ptr) {
```

Comment:
I think there should be an address space check here still. We don't need to decorate workgroup pointers for example and cannot decorate input or uniform constant.

Reply:
Done